### DISCUSSION TITLE
Any Plans to Bring Node v21 Fetch to Node v20 or v18?

### DISCUSSION THREAD

**Discussion by nsunga** [nodejs/help#4647] [Q&A] [answered]
In Node v21, Fetch was deemed as stable: https://nodejs.org/docs/latest-v21.x/api/globals.html#fetch

Up to Node v20, Fetch was deemed as experimental: https://nodejs.org/docs/latest-v20.x/api/globals.html#fetch

Historically, I think there has been times of bringing stable features in newer node versions back to older versions with LTS. Are there any plans to do this with Fetch in Node v20 and 18?

Or if there is some public road map, that also would help - Apologies, this is my first time getting involved with the Node community. So Im unfamiliar with resources that are out there already

**Comment by marco-ippolito** [comment, marked as answer]
Hi @nsunga in a month the new LTS node 22 will be out 😃
Node 18 will be in maintance mode so its not going to be updated.
I dont know about node 20

**Reply by nsunga** [reply]
Awesome! Thank you for the heads up!
